lib/model/rofolder.go

```diff
@@ -20,7 +20,7 @@ func init() {
 
 type sendOnlyFolder struct {
 	folder
-	cfg config.FolderConfiguration
+	config.FolderConfiguration
 }
```

Comment:
There was a joke about how Latvian president asked Lithuanian president to lend them tanks when Russian invaded Latvia.

The Lithuanian president asked whether the Latvian president would like one of them, or all of them... while Lithuania only had two...

TL;DR:
Given all of our two folders are embedding `folder`, I suggest we embed config in `folder` instead.